src/components/Nav/Login/Login.js

```diff
@@ -0,0 +1,204 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import theme from '../../../styles/theme';
+import { flexAlignCenter } from '../../../styles/mixin';
+
+const Login = props => {
+  const { modalState, closeModal } = props;
+  const [userData, setUserData] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setUserData(prev => ({ ...prev, [name]: value }));
+  };
+
+  const disabled = userData.id.length * userData.pw.length > 1 ? false : true;
```

Comment:
```suggestion
  const disabled = !(userData.id.length * userData.pw.length > 1);
```
true or false 등 boolean이 필요한 경우에는 굳이 삼항연산자 쓸 필요 없습니다! 위와 같이 표현할 수 있을 것 같네요